axiom/tests/test_php_packages.c

```diff
@@ -203,4 +261,5 @@ void test_main(void* p NRUNUSED) {
   test_php_packages_to_json();
   test_php_package_exists_in_hashmap();
```

Comment:
Would be good to have a composer package added followed by composer package added for completeness.
`test_php_package_exists_in_hashmap` only tests legacy package followed by legacy package and `test_php_package_priority` only tests mixed packages.

Reply:
Does 0ab342b51ed1be5f93331e182caa8669b81af2c1 address your concern?